MONGODB/F1.3.md

```diff
@@ -0,0 +1,34 @@
+# F1.2: Introduction to Stitch + Using Stitch
+
+Another component of MongoDB is using Stich, which allows you to configure rules, authentication and other services for your services. To fully understand Stitch, let's build a blog which allows you to post and edit comments. 
```

Comment:
explain rules, authentication and other services before diving into making a blog